content/en/docs/concepts/architecture.md

```diff
@@ -6,7 +6,7 @@ aliases:
 - /docs/architecture/
 ---
 
-The following diagram illustrates the general system deployed on top of a kubernetes cluster.
+The following diagram illustrates the general system deployed on top of a Kubernetes cluster.
```

Comment:
Technically speaking, kubernetes is all lowercase. Their font is ubuntu, and you can see it there.